Mondee Watch App/View/ArrowAnimation.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  ArrowAnimation.swift
+//  Mondee Watch App
+//
+//  Created by 하명관 on 2023/07/13.
+//
+
+import SwiftUI
+
+struct ArrowAnimation: View {
+    @State var fade: Double = 0
+    @State var isAnimating: Bool = false
```

Comment:
타입 추론이 되기 때문에 타입 선언을 생략하는건 어떤가요?
```suggestion
    @State var fade = 0.0
    @State var isAnimating = false
```